collections/internet-censorship-circumventions/index.md

```diff
@@ -0,0 +1,13 @@
+---
+items:
+ - Dreamacro/clash
+ - Psiphon-Inc/psiphon
+ - getlantern/lantern
+ - shadowsocks/shadowsocks
```

Comment:
This repository is currently empty. Should we remove it from this list?
```suggestion
```

Reply:
Please see https://en.wikipedia.org/wiki/Shadowsocks#Takedown

Reply:
Thanks for the context, however, this is pointing to a GitHub repository that hasn't had any commits in 6 years, so if the development is still happening, it's happening somewhere else. Perhaps it would better to link to this wikipedia page instead? (see here for any url support: https://github.com/github/explore/blob/main/docs/API.md#items)
```suggestion
 - https://en.wikipedia.org/wiki/Shadowsocks
```

Reply:
there is no code in this repository, there is releases to download the previous code.

Reply:
Fair enough. Thanks for the discussion, let's leave this repo in then 👍🏼